lib/lita/handlers/github_prs/git_diff.rb

```diff
@@ -32,8 +32,16 @@ def pull_requests
         private
 
         def pull_request_commit?(commit)
+          merge_commit?(commit) || squash_merge_commit?(commit)
+        end
+
+        def merge_commit?(commit)
           commit.message.start_with?('Merge pull request')
         end
+
+        def squash_merge_commit?(commit)
+          commit.message.match(%r{\(#\d+\)$})
```

Comment:
What does the commit messages for Squash and Merge look like? I feel like this is a pretty liberal matcher.

Reply:
It'll take the title of the pr and add " (#[PR NUMBER])" as a suffix. 

Within GitHub that will translate to a url to the PR